src/workspaces/graphql/queries.graphql

```diff
@@ -0,0 +1,44 @@
+query WorkspacesPage($page: Int, $perPage: Int) {
+  workspaces(page: $page, perPage: $perPage) {
+    totalItems
+    items {
+      id
+      name
+      countries {
+        code
+        flag
+      }
+    }
+  }
+}
+
+query WorkspacePage($id: String!, $page: Int, $perPage: Int) {
+  workspace(id: $id) {
+    id
+    name
+    description
+    countries {
+      code
+      flag
+      name
+    }
+    permissions {
+      delete
+      update
+      manageMembers
+    }
+    members(page: $page, perPage: $perPage) {
```

Comment:
Where is it needed ?

Reply:
On the workspace members list

Reply:
Ok I see what you did :p. You have to create another query for this component. (you can add it directly in the component `WorkspaceMembers`)